public/style.css

```diff
@@ -648,10 +654,6 @@ body {
   background: #f8f8f8;
 }
 
-#room input, #room textarea {
-  font-size:1.1em;
-}
-
```

Comment:
You removed this which reverts [this](https://github.com/KamandPrompt/baat-cheet/commit/dcb5027d468cbfe5e50267428e1ff30abfa244c1).

Reply:
I fixed this and did a `squash`. Let me know if it helps or not.